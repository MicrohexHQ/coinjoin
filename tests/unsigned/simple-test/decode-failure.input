010000000118d4fbf6507bb2b6623ab63baa92c5cee90343a690da098cbcec41eeb5539aad0d00000000ffffffff0100e1f505000000001976a914be97dbc413610be87c0d3fee05a9aeda548183c088ac

