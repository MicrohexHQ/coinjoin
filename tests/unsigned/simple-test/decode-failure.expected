mpo: 
mpc: 
hex: 
err: Failedtomergetransactions.
